cypress.json

```diff
@@ -0,0 +1,5 @@
+{
+        "baseUrl": "http://localhost:1234",
```

Comment:
I think you need to run `prettier` again here 🤔

Reply:
Isn't the husky hook running the linting and prettier application wide?

Reply:
I checked it is not running on Husky hooks and apparently the root files are being ignored by Husky.